src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PremisesEntity.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.OneToMany
+import javax.persistence.Table
+
+@Entity
+@Table(name = "premises")
+data class PremisesEntity(
+  @Id
+  var id: UUID,
+  var name: String,
+  var apCode: String,
+  var postcode: String,
+  @OneToMany(mappedBy = "premises")
+  var beds: MutableList<BedEntity>,
```

Comment:
At this point, we won't be modelling the concept of beds (just a total bed count), so this might be a premature optimisation.

Reply:
Have switched it to just be a total count column on the Premises table